web_monitoring/db.py

```diff
@@ -489,7 +500,7 @@ def add_versions(self, versions, *, update='skip', create_pages=None,
             If true, don't import versions of a page that have the same hash as
             the version captured immediately before them.
         batch_size : integer, optional
-            Default batch size is 50000 Versions.
+            Default batch size is 1000 Versions.
```

Comment:
This looked out of date based on the signature above:
```
    def add_versions(self, versions, *, update='skip', create_pages=None,
                     skip_unchanged_versions=None, batch_size=1000):
```

Reply:
Ah, thanks for catching this. 50,000 is even above the [maximum size](https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/d20d4ed9be6e7214a49895582c17ff5de05b3296/app/controllers/concerns/paging_concern.rb#L4-L5)!